utils/seo/main.js

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
JSDocs seems not matching exported function (no product and url params support) also should be above function.

Reply:
Also, I would suggest to move this file to `config/head.js` (upcoming nuxt3 convention) and simply export head instead of a function (there is no need/customization)

Reply:
> JSDocs seems not matching exported function (no product and url params support) also should be above function.

Oh, I copied this code from one previous code. I remembered fixing it.

Reply:
> Also, I would suggest to move this file to `config/head.js` (upcoming nuxt3 convention) and simply export head instead of a function (there is no need/customization)

OK, Pooya
Thank you very much for the suggestion.